modules/persistent-storage-cinder-provisioning.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-cinder.adoc
+
+[id="persistent-storage-cinder-provisioning_{context}"]
+= Dynamic provisioning with Cinder
```

Comment:
I believe the following steps all detail manual provisioning, not dynamic, of volumes.

Reply:
Changing header to "Manual provisioning with Cinder"